holoviews/plotting/mpl/comms.py

```diff
@@ -13,8 +13,8 @@
 
 mpl_msg_handler = """
 /* Backend specific body of the msg_handler, updates displayed frame */
-target = $('#fig_{comm_id}');
-img = $('<div />').html(msg);
+var target = $('#fig_{plot_id}');
+var img = $('<div />').html(msg);
```

Comment:
Ouch! This could have gone badly wrong I think!

Reply:
Yes, making sure all the ids match up was the hardest part of this PR. If you want to do some testing I've got a notebook with all the various combinations of widgets/comms/backends.

Reply:
Sounds horrible. I'll pass, thanks! ;-p